Userland/Utilities/pls.cpp

```diff
@@ -28,40 +28,45 @@ bool unveil_paths(const char*);
 // Unveil paths, given the current user's path and the command they want to execute
 bool unveil_paths(const char* command)
 {
-    bool did_unveil_ok = false;
+    // Unveil all trusted paths with browse permissions
+    auto trusted_directories = Array { "/bin", "/usr/bin", "/usr/local/bin" };
+    for (auto directory : trusted_directories) {
+        if (unveil(directory, "b") < 0) {
+            perror("unveil");
+            return false;
+        }
+    }
```

Comment:
not for this PR, just curious: this means that `pls` will not find binaries in custom `PATH` directories right now as those are never `unveil()`'d with browse permissions, correct?

Reply:
Correct. The user must provide the full path to the executable if the executable resides outside the trusted system paths.